db/migrate/20120417190549_add_details_to_measurement_import.rb

```diff
@@ -6,6 +6,8 @@ def change
 
     add_column :measurement_imports, :orientation, :string
 
+    add_column :measurement_imports, :subtype, :string
```

Comment:
When changing the database one shall not add code to an old migration, because when deploying, Rails looks at the `schema_migrations` table to determine what migrations to run; consequently this new line will not have any effect on an existing production database waiting to be migrated.

The correct approach is to add another migration. Please refer to [Active Record Migrations](http://edgeguides.rubyonrails.org/active_record_migrations.html).